### ISSUE TITLE
How to use this vagrant plugin? Is there any configuration needed in Vagrantfile?

### ISSUE THREAD

**Issue by pedrofurtado** [fnordfish/vagrant-vbguest-centos#1]

**Comment by fnordfish** [comment]
Additionally to `vbguest`, you'll just `vagrant plugin install vagrant-vbguest-centos`
It should be picked up by vbguest automatically. However, when in doubt, you can manually force it's usage (as described in https://github.com/dotless-de/vagrant-vbguest#advanced-usage) like this:

```ruby
Vagrant::Config.run do |config|
  config.vbguest.installer = VagrantVbguestCentos::Installer
end
```

Please keep in mind, that this thing is quite old, experimental and probably does not what you are hoping for (it won't upgrade your kernel).

**Comment by fnordfish** [comment]
short info, that in the meantime there was a PR merged over at dotless-de/vagrant-vbguest#314 and just released as 0.17.0.beta2 if you want to try that out. You should however `vagrant plugin uninstall vagrant-vbguest-centos` first.
